private/protocol/jsonrpc/unmarshal_error.go

```diff
@@ -50,18 +70,33 @@ func (u *UnmarshalTypedError) UnmarshalError(
 	code := codeParts[len(codeParts)-1]
```

Comment:
right here, why cant we override this if the querycompatibility trait is present?

Reply:
Not sure what you mean, we are deriving query compatibility information with the presence of `queryExceptions`